From c6f727f08b4ebc8ebc46abe374345937a1b0f0b9 Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Sun, 12 Jan 2020 03:53:28 +0100
Subject: [PATCH] get rid of the vendored-openssl insanity

Using vendored openssl will break multiple targets because the
openssl-src crate only has a bunch of target triples defined.
The point of this change was to apparently simplify building
on musl, well, it worked on musl fine before and now it's broken.

This reverts https://github.com/EmbarkStudios/cargo-deny/pull/67.
---
 Cargo.toml | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git Cargo.toml Cargo.toml
index 566b02a..da3c844 100644
--- Cargo.toml
+++ Cargo.toml
@@ -42,7 +42,7 @@ parking_lot = "0.10.0"
 petgraph = "0.4.13"
 rayon = "1.2.1"
 regex = { version = "1.3.1", default-features = true }
-rustsec = { version = "0.16.0", features = ["vendored-openssl"] }
+rustsec = "0.16.0"
 semver = "0.9.0"
 serde = { version = "1.0.103", features = ["derive"] }
 serde_json = "1.0.44"
@@ -60,6 +60,3 @@ lazy_static = "1.4.0"
 # We use this for creating fake crate directories for
 # crawling license files on disk
 tempfile = "3.1.0"
-
-[patch.crates-io]
-rustsec = { git = "https://github.com/RustSec/rustsec-crate.git", rev = "aaba369" }
-- 
2.24.0

